### PR TITLE
Switch to previous desktop

### PR DESCRIPTION
**Pull request by Perlence**
Remember the number of last desktop for quick switch.

### CODE REVIEW

File: desktop_switcher.ahk [hunk 1]
```diff
@@ -1,15 +1,14 @@
 ; Globals
-DesktopCount = 2        ; Windows starts with 2 desktops at boot
-CurrentDesktop = 1      ; Desktop count is 1-indexed (Microsoft numbers them this way)
+global DesktopCount := 4        ; Windows starts with 2 desktops at boot
```
Comment (pmb6tz) [inline desktop_switcher.ahk]: Why 4 desktops here? by default windows starts with 2

Comment (Perlence) [inline desktop_switcher.ahk]: Oh, sorry, just my setup leaked.

**Comment by Elijas** [comment]
Thank you, that would be a great idea. I personally use this in my own set up a lot :)

However, flags in function arguments are considered an anti-pattern. If I'll have time, I'll try to fix the merge conflicts.

**Comment by pmb6tz** [comment]
This issue is fixed now. Thanks!

**Pull request closed without merging**